content/blog/2022-12-06-keda-2.9.0-release.md

```diff
@@ -0,0 +1,39 @@
++++
+title = "Announcing KEDA v2.9 🎉"
+date = 2022-12-08
+author = "Jeff Hollan (KEDA), Tom Kerkhove (KEDA)"
+aliases = [
+"/blog/2.9.0-release"
+]
++++
+
+We recently completed our most recent release: 2.9.0 🎉!
+
+Here are some highlights:
+
+- A pre-made Grafana dashboard to visualize metrics exposed by the KEDA metrics adapter ([docs](https://keda.sh/docs/2.9/operate/prometheus/))
+- New Loki Scaler ([docs](https://keda.sh/docs/scalers/loki/))
+- All metrics exposed to prometheus are now consolidated in KEDA Operator
+- Azure Key Vault now suppoerts pod identities for authentication
+
+Potential breaking changes include:
+- Upgrading the API version of managed HPA objects from `autoscaling/v2beta2` to `autoscaling/v2`
+
+There are no deprecations to announce as part of this release.
+
+Let's have a closer look.
+
+## New & Improved Scalers
+
+KEDA now has 50+ built-in scalers, giving you instant access to provide event driven scale across a variety of sources. New with this release are:
+- Loki ([docs](https://keda.sh/docs/scalers/loki/))
+
+In addition to new scalers, various new features and improvements were added to Kafka, Azure Event Hubs, Azure Pipelines, GCP Storage, Metrics, NATS Jetstream, Prometheus, Pulsar, Azure Blob, GCP Stackdriver, and New Relic scalers. Details on the underlying fixes can be found in the [changelog](https://github.com/kedacore/keda/blob/main/CHANGELOG.md#v290).
+
+## And more...
```

Comment:
Maybe we can add a section on our improved Prometheus metrics and the pre-made dashboard?

Reply:
Checking to get screenshot of latest dashboard version [here](https://github.com/kedacore/keda/pull/3934#issuecomment-1340608964) otherwise we can use the one in our docs PR [here](https://github.com/kedacore/keda-docs/pull/985)